### PR TITLE
Send errors to webhook

### PR DESCRIPTION
**Pull request by cfpwastaken**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -8,9 +8,37 @@ import { COLOR } from "./logger.js"
 import points from "./points.js";
 import { db, createTables } from "./sql.js";
 
+export const webhook = new WebhookClient({ url: config.webhook });
+
```
Comment (j0code) [inline main.js:12]: make it only do that webhuk stuff when config.webhook is defined
(this also counts for the process.on listeners)

**Review by j0code — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (cfpwastaken) [reply main.js]: did

File: main.js [hunk 2]
```diff
@@ -8,37 +8,40 @@ import { COLOR } from "./logger.js"
 import points from "./points.js";
 import { db, createTables } from "./sql.js";
 
-export const webhook = new WebhookClient({ url: config.webhook });
+export let webhook = null;
```
Comment (j0code) [inline main.js:11]: can u pls rename it to errorhook? webhook could be anything

**Review by j0code — CHANGES_REQUESTED** [review]

File: main.js (hunk 2 above)
Comment (cfpwastaken) [reply main.js]: done

**Review by j0code — APPROVED** [review]
luks gud
seal-approval

**Pull request merged**